src/main/java/Main/Main.java

```diff
@@ -24,13 +33,26 @@ public static void main(String[] args) {
         DatabaseController databaseInteractor = new DatabaseController();
 
         //Open the application with the Login Page
-        currentPage = new LoginPage();
+
+        // setup test currUser
```

Comment:
again we dont need the code of u testing the listing (this should not have been added to git add.)

Reply:
fixed